packages/zoe/docs/zoe-catalog-of-storage.md

```diff
@@ -0,0 +1,217 @@
+# Catalog of Zoe Maps/WeakMaps/Stores
+
+## Zoe Service (one per chain)
+
+### installations - native WeakSet
+
+An installation is added whenever a user calls `E(zoe).install` to
+install a contract bundle. 
+
+**Expected cardinality**: one element per call
+to `E(zoe).install`. On the testnet with only the bootstrap and load
+generation code running, this should be about 5 elements total. 
+
+**Access patterns**: We `add` and `has` only.
```

Comment:
```suggestion
**Access patterns**: We use `add` and `has` only.
```

use?  send?   `get`, `add`, and `init` read acceptably, but `has` has an odd feel to it.  
I think I'd prefer
```
**Access patterns**:`add`, `has`.
```

This is a nit, since the doc is internal only. Treat it as no more than a suggestion.

Reply:
That's a good idea.